### PR TITLE
[Telemetry Tables] e2e Test to make sure changing sort order in limited mode makes another request

### PR DESCRIPTION
**Pull request by jvigliotta**
Closes 

### Describe your changes:

Add test to make sure changing sort order in limited mode triggers a another request for telemetry.

Covers this change: https://github.com/nasa/openmct/pull/7875
Closes: https://github.com/nasa/openmct/issues/7876

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [ ] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [ ] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: tests/e2e/yamcs/telemetryTables.e2e.spec.mjs [hunk 1]
```diff
@@ -79,6 +79,30 @@ test.describe("Telemetry Tables tests @yamcs", () => {
         await expect(page.getByRole('button', { name: 'SHOW LIMITED' })).toBeVisible();
     });
 
+    test('Changing sort order in limited mode triggers a new request', async ({ page }) => {
+        // Set up request promise for an events request in descending order
+        const eventRequestOrderDescending = page.waitForRequest(/.*\/api\/.*\/events.*order=desc$/);
+
+        // Navigate to the Events table
+        await page.getByLabel('Navigate to Events yamcs.').click();
```
Comment (unlikelyzero) [inline tests/e2e/yamcs/telemetryTables.e2e.spec.mjs:87]: this works, but 

```
await page.goto(./events, { waitUntil: 'networkidle' }) 
```

**Review by unlikelyzero — APPROVED** [review]
Looks good. Alternative suggestion in place

File: tests/e2e/yamcs/telemetryTables.e2e.spec.mjs [hunk 2]
```diff
@@ -84,7 +84,7 @@ test.describe("Telemetry Tables tests @yamcs", () => {
         const eventRequestOrderDescending = page.waitForRequest(/.*\/api\/.*\/events.*order=desc$/);
 
         // Navigate to the Events table
-        await page.getByLabel('Navigate to Events yamcs.').click();
+        await page.goto('./events', { waitUntil: 'networkidle' });
         await page.waitForLoadState('networkidle');
```
Comment (unlikelyzero) [inline tests/e2e/yamcs/telemetryTables.e2e.spec.mjs:88]: so you can remove this line now

Comment (jvigliotta) [reply tests/e2e/yamcs/telemetryTables.e2e.spec.mjs:88]: wow, how did I miss that? will do!

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**